### PR TITLE
Update gophercloud, enable multiattach functionality

### PR DESCRIPTION
**Pull request by jamesmmccarthy**

### CODE REVIEW

**Comment by ozerovandrei** [comment]
@jamesmmccarthy Hello.

Usually we're updating dependencies in a separate commit that is part of a PR that implements or fixes some issue.
You can see some recently closed PRs as a reference: #409 #421.

As far as I understand, you want to implement multiattach support in the `terraform-provider-openstack`?

Can you please create an issue explaining it?
Also, it's better to rename this PR and add all needed commits that implement resource or datasource in it.

**Comment by jtopjian** [comment]
@ozerovandrei for some background, James and I have been chatting about this offline. I explained poorly that the vendor commit should be part of the PR which includes multiattach support. :)

**Comment by jamesmmccarthy** [comment]
Issue: Feature request: Update Openstack Provider multiattach functionality #432

**Comment by jtopjian** [comment]
@jamesmmccarthy The vendor commit itself looks good!

I also see you've renamed the title and such. So just append a second commit with the multiattach support and that should be it :)

**Comment by jamesmmccarthy** [comment]
Ah yes ! Ok, I need to look into this a bit more before adding follow on commits - change to WIP

**Comment by jamesmmccarthy** [comment]
remove WIP, ready for review

File: openstack/resource_openstack_blockstorage_volume_v3_test.go [hunk 1]
```diff
@@ -89,6 +89,26 @@ func TestAccBlockStorageV3Volume_image(t *testing.T) {
...
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckBlockStorageV3VolumeDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccBlockStorageV3Volume_image_multiattach,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckBlockStorageV3VolumeExists("openstack_blockstorage_volume_v3.volume_1", &volume),
+					resource.TestCheckResourceAttr(
```
Comment (ozerovandrei) [inline openstack/resource_openstack_blockstorage_volume_v3_test.go]: Can we also check `multiattach` attribute here?

Comment (jamesmmccarthy) [reply openstack/resource_openstack_blockstorage_volume_v3_test.go]: Ah yes ! Well spotted, I meant to include that - I'll do these updates shortly

**Comment by jamesmmccarthy** [comment]
I may not have resolved the issue correctly in the UI here - please re-review :)

File: website/docs/r/blockstorage_volume_v3.html.markdown [hunk 2]
```diff
@@ -64,6 +64,9 @@ The following arguments are supported:
 * `volume_type` - (Optional) The type of volume to create.
     Changing this creates a new volume.
 
+* `multiattach` - (Optional) When this option is set it allows the volume to
```
Comment (ozerovandrei) [inline website/docs/r/blockstorage_volume_v3.html.markdown]: Let's rephrase it:
`Allow the volume to be attached to more than one Compute instance.`

**Comment by jamesmmccarthy** [comment]
Thanks for the feedback ! Ok did some quick updates there

**Comment by jamesmmccarthy** [comment]
recheck

**Pull request merged**